src/server/utils.ts

```diff
@@ -33,9 +35,29 @@ export function shouldSkip(
     return skipOptions;
   }
   if (Array.isArray(skipOptions)) {
-    return skipOptions.map((skipOption) => shouldSkip(browser, meta, skipOption, test)).find(Boolean) || false;
+    for (const skip of skipOptions) {
+      const reason = shouldSkip(browser, meta, skip, test);
+      if (reason) return reason;
+    }
+    return false;
+  }
+  let hasSkipOptionKeys = false;
+  for (const skipKey in skipOptions) {
+    if (skipOptionKeys.includes(skipKey)) {
+      hasSkipOptionKeys = true;
+      continue;
+    }
```

Comment:
If we keep both "object" formats (the old one with "skipOptionKeys" and the new one with "reason keys") then users will be able to use different formats on different levels. It will lead to unexpected result as the old one will overlap the new despite the level. I think we should avoid this somehow.

Reply:
Options in old and new formats will be merged and checked. The only one exception, when you combine new format with an array, so depending on level array or new format will be overridden

Reply:
Yeah, I was wrong, everything works fine.